src/python_testing/TC_OPSTATE_2_1.py

```diff
@@ -125,26 +125,22 @@ async def test_TC_OPSTATE_2_1(self):
             in_range = (0x80 <= operational_state and operational_state <= 0xBF)
             asserts.assert_true(operational_state in defined_states or in_range, "OperationalState has an invalid ID value!")
 
-            if 'PIXIT_OPSTATE_ST_STOPPED' in self.matter_test_config.global_test_params:
-                if self.matter_test_config.global_test_params['PIXIT_OPSTATE_ST_STOPPED'] == 1:
-                    self.print_step("6a", "Manually put the device in the stopped state")
-                    input("Press Enter when done.\n")
-                    await self.read_and_validate_opstate(step="6b", expected_state=Clusters.OperationalState.Enums.OperationalStateEnum.kStopped)
-            if 'PIXIT_OPSTATE_ST_RUNNING' in self.matter_test_config.global_test_params:
-                if self.matter_test_config.global_test_params['PIXIT_OPSTATE_ST_RUNNING'] == 1:
-                    self.print_step("6c", "Manually put the device in the running state")
-                    input("Press Enter when done.\n")
-                    await self.read_and_validate_opstate(step="6d", expected_state=Clusters.OperationalState.Enums.OperationalStateEnum.kRunning)
-            if 'PIXIT_OPSTATE_ST_PAUSED' in self.matter_test_config.global_test_params:
-                if self.matter_test_config.global_test_params['PIXIT_OPSTATE_ST_PAUSED'] == 1:
-                    self.print_step("6e", "Manually put the device in the paused state")
-                    input("Press Enter when done.\n")
-                    await self.read_and_validate_opstate(step="6f", expected_state=Clusters.OperationalState.Enums.OperationalStateEnum.kPaused)
-            if 'PIXIT_OPSTATE_ST_ERROR' in self.matter_test_config.global_test_params:
-                if self.matter_test_config.global_test_params['PIXIT_OPSTATE_ST_ERROR'] == 1:
-                    self.print_step("6g", "Manually put the device in the error state")
-                    input("Press Enter when done.\n")
-                    await self.read_and_validate_opstate(step="6h", expected_state=Clusters.OperationalState.Enums.OperationalStateEnum.kError)
+            if self.check_pics("OPSTATE.M.ST_STOPPED"):
```

Comment:
```suggestion
            if self.check_pics("OPSTATE.S.M.ST_STOPPED"):
```

We need to use the "S" (server) name particle too.